js/main.js

```diff
@@ -25,7 +25,7 @@ $timeChoice.addEventListener('submit', getTime);
 function getTime(event) {
   event.preventDefault();
   time = parseInt($timeChoice.elements.time.value);
-  if (Object.is(time, NaN)) {
+  if (isNaN(time)) {
```

Comment:
Did you test your code after making this change? This will now only run if the value **_is_** NaN 👀 . You want to make us of a `!` operator at the beginning of the condition